services/src/northbound/src/main/java/org/openkilda/northbound/messaging/kafka/KafkaHealthCheckMessageConsumer.java

```diff
@@ -38,13 +39,18 @@
 import java.util.stream.Collectors;
 
 /**
- * Kafka Health-Check message producer.
+ * Kafka Health-Check message consumer.
  */
 public class KafkaHealthCheckMessageConsumer implements HealthCheckMessageConsumer {
     /**
      * Expected topologies count.
      */
-    private static final int HEALTH_CHECK_COMPONENTS_COUNT = 5;
+    private static final int HEALTH_CHECK_COMPONENTS_COUNT = ServiceType.values().length;
```

Comment:
This is no a solution.

* we don't need a separate tooling(KafkaHealthCheckMessageConsumer) for health check messages (see https://github.com/telstra/open-kilda/pull/978 how it should become in near future)
* current health check architecture produce too many false positive responses (any component timeout produce "unhealthy" state of whole system)
* current implementation is too sensitive to kafka timeouts

Prepare architecture proposition (sequence diagram), after it's approval it can be implemented.